Stream/DynamicDestinationMessaging/Startup.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using Steeltoe.Stream.Binding;
+
+namespace DynamicDestinationMessaging
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "DynamicDestinationMessaging", Version = "v1" });
+            });
+
+            services.AddTransient<BinderAwareChannelResolver>();
```

Comment:
This is fine that it is added to the DI container here, but was thinking we should scale it down to a GenericHost Console app without Startup file and then accomplish the routing via an incoming message rather than HTTP.